cfgov/cfgov/settings/production.py

```diff
@@ -185,8 +185,30 @@
 SESSION_EXPIRE_AT_BROWSER_CLOSE = True
 CSRF_COOKIE_HTTPONLY = True
 CSRF_COOKIE_SECURE = True
-SECURE_BROWSER_XSS_FILTER = True
 SECURE_HSTS_SECONDS = 600
 SECURE_HSTS_INCLUDE_SUBDOMAINS = True
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SECURE_HSTS_PRELOAD = True
+SECURE_SSL_REDIRECT = True
+
+# Require the SECRET_KEY as an environment variable
+try:
+    SECRET_KEY = os.environ["SECRET_KEY"]
+except KeyError:
+    raise ImproperlyConfigured(
+        "The SECRET_KEY environment variable must be set"
+    )
+
+# Secret key fallbacks that allow the rotation of the secret key.
+# This environment variable should be a JSON array if fallbacks are available.
+# This list is empty (no old fallback secret keys) if the environment variable
+# SECRET_KEY_FALLBACKS is not set.
+try:
```

Comment:
Would it be nice to unify this logic with the `ALLOWED_HOSTS` logic above, by creating a simple helper function? Would avoid some duplication.

Reply:
This is a very good idea, and would apply to `CORS_ALLOWED_ORIGINS` too!

Reply:
[Added](https://github.com/cfpb/consumerfinance.gov/pull/8196/files#diff-399c82f449c78fc61059580ebcc2436b6d010ee9b6e0d3e87603a53e1d86c770R18-R26) in eb373065854a345921e00d240d13cac788a1882f !